src/components/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { styled, ThemeProvider } from '@mui/material/styles';
+import theme from '../../styles/theme/theme';
+import { CircleIconButton } from '../CircleIconButton/CircleIconButton';
+import { Label } from '../Label/Label';
+import { Grid } from '@mui/material';
+import { Typography } from '@mui/material';
+import ArrowCircleRightIcon from '@mui/icons-material/ArrowCircleRight';
+import {
+  SurahLessonCard,
+  SurahLessonCardProps,
+} from '../SurahLessonCard/SurahLessonCard';
+
+export interface SideBarProps {
+  lessonCardsList: Array<{ lessonCardData: SurahLessonCardProps }>;
+}
+
+const SideBarGrid = styled(Grid)`
+  ${({ theme }) => `
+border-right: solid;
+border-width: 0.5px;
+border-color: #797979;
+background-color: inherit;
+`}
+`;
+
+export const SideBar = ({ lessonCardsList, ...props }: SideBarProps) => {
+  return (
+    <ThemeProvider theme={theme}>
```

Comment:
I'm curious, what does `ThemeProvider` do? does it basically set the CSS for the class?